modules/github.py

```diff
@@ -36,7 +36,7 @@ class Module(ModuleManager.BaseModule):
     @utils.hook("api.post.github")
     def github(self, event):
         payload = event["data"].decode("utf8")
-        if event["headers"]["content-type"] == FORM_ENCODED:
+        if event["headers"]["Content-Type"] == FORM_ENCODED:
```

Comment:
According to the Github Documentation it is sent as `Content-Type`

Reply:
I promise you it does not.

Reply:
In the following Code snippet from the GitHub documentation it is

```
POST /payload HTTP/1.1
Host: localhost:4567
X-Github-Delivery: 72d3162e-cc78-11e3-81ab-4c9367dc0958
X-Hub-Signature: sha1=7d38cdd689735b008b3c702edd92eea23791c5f6
User-Agent: GitHub-Hookshot/044aadd
Content-Type: application/json
Content-Length: 6615
```
https://developer.github.com/webhooks/

Reply:
And this is actual data that we get when it hits bitbot

````
{'Host': [redacted], 'Accept': '*/*', 'User-Agent': 'GitHub-Hookshot/ddae561', 'X-GitHub-Event': 'pull_request_review_comment', 'X-GitHub-Delivery': [redacted], 'content-type': 'application/json', 'X-Hub-Signature': [redacted], 'Content-Length': '22449'} 
````

Reply:
(those are the dict of headers)

Reply:
I suggested camel case earlier - I actually meant title case.

```
>>> "content-type".title()
'Content-Type'
```

Reply:
This works, perhaps for the wrong reasons, but the _fix_ will clearly break the feature entirely. Bugfixes are most welcome, but please please test to make sure you've actually found a bug

Reply:
Agree - this PR should be an enhancement to the REST API system (making header keys either case-insensitive or predictably capitalised) because the "bug" you were trying to fix was done intentionally and this pull request would actually introduce a bug where there wasn't one

Reply:
I would argue that the bug is on the Github end, as AFAIK all other sites use Content-Type. So while we should code around this bug, we should file a Bug report of some sort on this, as it breaks expected and (honestly mostly standard) behavior.